pkg/asset/agent/joiner/clusterinfo.go

```diff
@@ -688,3 +691,23 @@ func (ci *ClusterInfo) reportResult(ctx context.Context) error {
 
 	return workflowreport.GetReport(ctx).StageResult(workflow.StageClusterInspection, string(data))
 }
+
+func (ci *ClusterInfo) removeDuplicates(input string) string {
+        entries := strings.Split(strings.TrimSpace(input), ",")
+
+        uniqueMap := make(map[string]bool)
+        var uniqueList []string
+        for _, entry := range entries {
+                trimmedEntry := strings.TrimSpace(entry)
+
+                // Check if the entry is not empty and hasn't been added to the map yet
+                if trimmedEntry != "" {
+                        if _, exists := uniqueMap[trimmedEntry]; !exists {
+                                // If it doesn't exist, add it to the map and the unique list
+                                uniqueMap[trimmedEntry] = true
+                                uniqueList = append(uniqueList, trimmedEntry)
+                        }
```

Comment:
Otherwise a trace could be useful, not only for later troubleshooting but also to inform the user of a potential misconfiguration currently present in the Proxy resource